deploy/base/deployment.yaml

```diff
@@ -113,10 +109,6 @@ spec:
                 configMapKeyRef:
                   name: mixer-configmap
                   key: serviceName
-          resources:
-            limits:
-              memory: "0.5G"
-              cpu: "100m"
```

Comment:
How can we monitor the impact of this change?  Meaning, how often does this happen and how many other pods are affected in collateral damage.

Reply:
Need to setup monitoring on node/pod level mem/cpu usage and pod restart.